vue.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  chainWebpack: (config) => {
+    config
+      .plugin('html')
+      .tap(args => {
+        args[0].meta = {
+          viewport: 'width=device-width, initial-scale=1.0',
+          'theme-color': '#02002E',
+          description: process.env.VUE_APP_WEB_DESCRIPTION,
```

Comment:
Add keywords

Reply:
Google does not use the keywords meta tag in web ranking
https://webmasters.googleblog.com/2009/09/google-does-not-use-keywords-meta-tag.html